### DISCUSSION TITLE
What is "Status Assigned"? Can we still get assigned if the issue is already having the label "Status Assigned"

### DISCUSSION THREAD

**Discussion by aryanaarav** [abhisheks008/DL-Simplified#644] [General]
I opened the issues of the project and I see that there are a lot of labels listing "Status Assigned". Does this mean that the issue has been closed for new to people to take on?

**Comment by abhisheks008** [comment]
Status Assigned means someone has been assigned to that issue and is working on that. As per the code of conduct, one contributor can work on one issue at a time. You can check out the issues which are having the label as `Up for Grabs`.
